OWNERS_ALIASES

```diff
@@ -31,3 +31,31 @@ aliases:
     - dariadomagala
     - akucharska
     - sjanota
+  skydiving-tunas
+    - @Abd4llA
+    - @joek
+    - @abbi-gaurav
+    - @rakesh-garimella
+    - @lilitgh
+    - @k15r
+    - @nachtmaar
+    - @anishj0shi
+    - @montaro
+    - @marcobebway
+    - @radufa
+    - @sayanh
+    - @antoineco
+  skydiving-tunas
```

Comment:
twice twice?

Reply:
i see the other teams differentiate between reviewers and approvers. Shall we do the same even if they would be the same initially?
```suggestion
  skydiving-tunas-reviewers:
```


if we decide to do so all the OWNERS files need to be updated